### PR TITLE
[RELAY] MobileNet Port

### PR DESCRIPTION
**Pull request by eqy**
Pattern-matching port of NNVM MobileNet to Relay.
This is not yet deeply tested (e.g., using TVM Runtime or end2end evaluation).

### CODE REVIEW

**Comment by tqchen** [comment]
please request reviews

**Comment by eqy** [comment]
@Huyuwei please review

File: python/tvm/relay/testing/mobilenet.py [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+    Returns
+    -------
+    net : relay.Function
+        The computational graph
+
+    params : dict of str to NDArray
+        The parameters.
+    """
+    image_shape = tuple([batch_size] + list(image_shape))
```
Comment (joshpoll) [inline python/tvm/relay/testing/mobilenet.py]: Should this be moved inside `mobile_net`? `resnet` takes `batch_size` as an argument.

Comment (tqchen) [reply python/tvm/relay/testing/mobilenet.py]: @eqy +1, maybe we need to be consistent, either pass batch_size as a separate argument, or rename to data_shape

File: python/tvm/relay/testing/resnet.py [hunk 2]
```diff
@@ -221,15 +222,15 @@ def get_net(batch_size,
...
-    return resnet(units=units,
+    resnet1 = resnet(units=units,
                   num_stages=num_stages,
                   filter_list=filter_list,
                   num_classes=num_classes,
                   data_shape=data_shape,
                   bottle_neck=bottle_neck,
                   dtype=dtype)
-
+    print(resnet1)
```
Comment (tqchen) [inline python/tvm/relay/testing/resnet.py]: remove print

**Comment by tqchen** [comment]
needs rebase again from upstream/master

**Comment by tqchen** [comment]
Thanks @joshpoll @eqy this is now merged

**Pull request merged**